docs/build.md

```diff
@@ -420,3 +420,150 @@ npx hardhat compile
 Because you used TypeScript, the compile task also generates TypeScript bindings using [TypeChain](https://www.npmjs.com/package/typechain).
 
 
+## Build a dApp locally
+
+*Learn how to set up a local environment or a local Yuma testnet node to build a dApp (decentralized application).*
+
+In this section, you will accomplish the following:
+
+* Deploy an existing dApp into a local environment or Yuma Testnet
+* Run a private instance of EON 
+
+The easiest way to deploy a dApp is to use **Docker**. The EON client will be running in a Dockerized container. See **Node Client** below.
```

Comment:
Can we link `**Node Client**` to the section below : `/docs/build#node-client` ?